applications/post-editing.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Human editing of machine translation output
+---
+
+# Post-editing
+
+Human **post-editing** is the review and correction of machine translation output. When post-editing, the human translator checks the original input and the machine translation output, and fixes errors in the output.
+
+Post-editing started to replace human translation because it is faster and cheaper than the traditional human translation process.
+
+Over the 2010s, with rising machine translation quality, it became the dominant human translation workflow.
```

Comment:
Is this true by the way?

https://twitter.com/bittlingmayer/status/1404358919640780802

Reply:
There are two things here:
-Smaller LSPs don't (usually) invest in MT as part of their internal workflows.
-Over the 2010s, MLVs started implementing MT and requesting post-editing services from LSPs.
So, the sentence is true. However, maybe it's best to delete the timeframe.